clippy_config/src/conf.rs

```diff
@@ -645,6 +645,47 @@ define_Conf! {
     ///
     /// Whether to also emit warnings for unsafe blocks with metavariable expansions in **private** macros.
     (warn_unsafe_macro_metavars_in_private_macros: bool = false),
+    /// Lint: FLOAT_CMP
+    ///
+    /// Whether to ignore comparisons to a named constnat
+    ///
+    /// #### Example
+    /// ```no_run
+    /// const VALUE: f64 = 1.0;
+    /// fn is_value(x: f64) -> bool {
+    ///     // Will warn if the config is `false`
+    ///     x == VALUE
+    /// }
+    /// ```
+    (float_cmp_allowed_constants: Vec<String> = Vec::new()),
```

Comment:
I would suggest adding `"*"` as a magic value that says that all constants are allowed. THis should also be documented.

Reply:
This would be better done in a more uniform way across all the multi-value configs.